playbook/app/pb_kits/playbook/pb_card/_card_mixin.scss

```diff
@@ -52,7 +52,7 @@ $pb_card_header_colors: map-merge(map-merge($product_colors, $additional_colors)
   background-color: $white;
 
   @each $name, $color in $background_colors {
-    &[class*=background_#{$name}]:not([class^=hover]) {
```

Comment:
This bug is going to be fixed in [PLAY-1603](https://runway.powerhrg.com/backlog_items/PLAY-1603)